src/aws/names.js

```diff
@@ -1,7 +1,12 @@
 const util = require('util')
+const md5 = require('md5')
 
 function clean (input) {
-  return input.replace(/[^a-z0-9+]+/gi, '')
+  return turncate(input.replace(/[^a-z0-9+]+/gi, ''))
+}
+
+function turncate (input) {
+  return input.length > 64 ? md5(input) : input
```

Comment:
Suggestion: since `md5()` functions returns a hex string (length = 128 bit / 4 bit = 32 characters), and the roleName can be up to 64 characters long, would it be useful to generate a roleName by taking the first 32 characters from the original role name + 32 characters from the hex string? 

This helps provide a bit of context when there are lots of looking through the list of Roles in IAM within the account when there are potentially lots of serverless deployments.

Just a thought. 😃